EdFi.Buzz.Installer/eng/README.md

```diff
@@ -60,36 +71,51 @@ You can configure the server, credentials or name to connect to the Sql Server O
 #### etl
 
 Options to configure the ETL to load from the database or file to the postgres database.
+
+![ETL](./images/etl.png)
+
 - **datasourceFormat:** If the data source corresponds to the Analytics Middle Tier (amt) or direct views of the ODS tables. Allowed values: amt, ods.
 - **odsDataStandard:** Database standard to use, if it is Data Standard 2 or 3.x. Allowed values: ds2, ds3.
 
 #### api
+
+![API](./images/api.png)
+
 - **version:** The NuGet version to download. Blank gets latest.
 - **url:**  URL for the GraphQL endpoint setting in the UI env file.
 - **port:** API port.
 
 #### ui
+
+![UI](./images/UI.png)
+
 - **version:** The NuGet version to download. Blank gets latest.
 - **port:** Port to access UI.
 - **externalLogo:** If true, look for images in an external URL. If false, the images must have been copied into the 'assets' folder of the UI. 
 - **logo:** Login page logo URL.
 - **logoWidth:** Logo max width.
 - **title:** Site title.
 - **titleLogo:** Site logo title to display in header URL.
-- **titleLogoWidth: Site logo title max width.
+- **titleLogoWidth:** Site logo title max width.
 - **titleLogoHeight:** Site logo max height.
 
 #### Change UI Web App icon (fav.ico)
-To update the icon displayed in the browser, it is required to replace the fav.ico file with a valid icon file. You must copy the fav.ico that you want to use in the web application, in the public folder.
+To update the icon displayed in the browser, it is required to replace the fav.ico file with a valid icon file. You must copy the fav.ico that you want to use in the web application, in the public folder (**EdFi.Buzz.UI\public\fav.ico**).
```

Comment:
This is the project location. The installed file location will be in the C:\Ed-Fi\Buzz\UI\fav.ico, by default.